source/addComment.js

```diff
@@ -12,4 +12,23 @@ $(document).ready(function(){
       </form>
     `)
   })
+
+  $('#comments').on('click', '#submit-comment-button', () => {
+    event.preventDefault();
+    const comment = $('textarea[name=comment]').val();
+    const author = $('input[name=author]').val();
+
+
+    comment_template = `
+      <ul id="comment_list">
```

Comment:
This `comment_template` could be its own function that takes `comment` as an argument. This would tidy up this function a bit.